### DISCUSSION TITLE
getting error message 'NoneType' object has no attribute 'DestroyTimer' when using update function

### DISCUSSION THREAD

**Discussion by andersresell** [pyvista/pyvista#3711] [Q&A] [answered]
I'm getting this error message in several test cases, and I know that the update function of the Plotter class triggers it. I pulled the following code snipped off the documentation which triggers this error:
`import os
from tempfile import mkdtemp
import pyvista
from pyvista import examples
filename = examples.download_dual_sphere_animation(load=False)
reader = pyvista.PVDReader(filename)

gif_filename = os.path.join(mkdtemp(), 'dual_sphere.gif')

plotter = pyvista.Plotter()
plotter.open_gif(gif_filename)
for time_value in reader.time_values:
    reader.set_active_time_value(time_value)
    mesh = reader.read()
    _ = plotter.add_mesh(mesh, smooth_shading=True)
    _ = plotter.add_text(f"Time: {time_value:.0f}", color="black")
    plotter.write_frame()
    plotter.clear()
    plotter.enable_lightkit()
plotter.close()`

This produces the following error:

Traceback (most recent call last):
  File "test2.py", line 17, in <module>
    plotter.write_frame()
  File "/home/anders/miniconda3/envs/myenv/lib/python3.8/site-packages/pyvista/plotting/plotting.py", line 4365, in write_frame
    self.update()
  File "/home/anders/miniconda3/envs/myenv/lib/python3.8/site-packages/pyvista/plotting/plotting.py", line 1943, in update
    self.right_timer_id = self.iren.create_repeating_timer(stime)
  File "/home/anders/miniconda3/envs/myenv/lib/python3.8/site-packages/pyvista/plotting/render_window_interactor.py", line 780, in create_repeating_timer
    self.interactor.DestroyTimer(timer_id)
AttributeError: 'NoneType' object has no attribute 'DestroyTimer'

Anyone know what causes this? In general I've had a lot of trouble getting pyvista and pyvistaqt up and running on ubuntu. I want to be able to create animations, not only static figures.

**Comment by dcbr** [comment, marked as answer]
This is a known issue and will be fixed in an upcoming release (see #3617).
